docs/tags/common-schema.md

```diff
@@ -24,6 +24,11 @@ Refer to this list of default fields for Logs Apps. The tags below are applicabl
 | error.type | A reserved field for errors |
 | span.id | Building block of a trace in distributed tracing |
 | trace.id | Building block of a trace in distributed tracing |
+| geoip | An object reserved for location related data |
+| geoip.location | A latitude and longitude defining the location associated with the log line |
+| geoip.info.country | The country associated with the log line |
+| geoip.info.continent | The continent associated with the log line |
+| geoip.info.city | The city associated with the log line |
```

Comment:
May want to push this one up 2 rows.  city -> country -> continent.  Size/ordering thing.  OCD? ;)

Reply:
Good point, adjusted.